code/game/objects/structures/bounty_post.dm

```diff
@@ -0,0 +1,268 @@
+#define BOUNTY_COOLDOWN 10 MINUTES
+#define BOUNTY_MARKER "bounty marker"
+#define EASY_BOUNTY_MULTIPLIER 1.25
+#define MEDIUM_BOUNTY_MULTIPLIER 1.5
+#define HARD_BOUNTY_MULTIPLIER 2
+
+GLOBAL_LIST_EMPTY(holy_contracts)
+GLOBAL_LIST_EMPTY(contract_points)
+GLOBAL_LIST_EMPTY(assigned_bounties)
+GLOBAL_LIST_EMPTY(bounty_points_tracker)
+
+GLOBAL_LIST_INIT(possible_monsters, list(
+	/mob/living/basic/mining/ice_whelp,
+	/mob/living/basic/mining/ice_demon,
+	/mob/living/basic/mining/lobstrosity,
+))
+////items we can buy
+/datum/bounty_store_item
+	///typepath of our item
+	var/obj/item/item_path
+	///price of our item
+	var/item_price
+
+/datum/bounty_store_item/legion_core
+	item_path = /obj/item/organ/internal/monster_core/regenerative_core/legion
+	item_price = 100
+
+/datum/bounty_store_item/plasma_flame
+	item_path = /obj/item/crusher_trophy/plasma_flame
+	item_price = 1500
+
+/datum/bounty_store_item/kinetic_cosmetic
+	item_path = /obj/item/borg/upgrade/modkit/crystal
+	item_price = 500
+
+/datum/bounty_store_item/godslayer_blade
+	item_path = /obj/item/divine_blade
+	item_price = 10000
+
+/datum/holy_bounty
```

Comment:
This should have a class level doc because the typepath doesn't explain much on its own